### PR TITLE
5.1

### PR DESCRIPTION
**Pull request by jellespijker**
See full list of PR's here https://github.com/Ultimaker/Cura/pull/12708

We're basically adding support for Conan, which will copy all materials in a package such that we can install it easily in Cura

We need to add two secrets to this repository. That ensure that we get a message on the Slack channel `cura-build` when a new Conan package is enabled.

Can somebody with admin access contact me

### CODE REVIEW

**Review by Joeydelarago — APPROVED** [review]

File: .github/workflows/conan-package.yml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        branches:
+            - main
+            - 'CURA-*'
+            - '[1-9]+.[0-9]+'
+        tags:
+            - '[1-9]+.[0-9]+.[0-9]+'
+
+jobs:
+    conan-recipe-version:
+        uses: ultimaker/cura/.github/workflows/conan-recipe-version.yml@CURA-9365
```
Comment (Joeydelarago) [inline .github/workflows/conan-package.yml:26]:
```suggestion
        uses: ultimaker/cura/.github/workflows/conan-recipe-version.yml@main
```

File: .github/workflows/conan-package.yml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+        with:
+            project_name: fdm_materials
+
+    conan-package-export-linux:
+        needs: [ conan-recipe-version ]
+        uses: ultimaker/cura/.github/workflows/conan-recipe-export.yml@5.1
```
Comment (Joeydelarago) [inline .github/workflows/conan-package.yml:32]:
```suggestion
        uses: ultimaker/cura/.github/workflows/conan-recipe-export.yml@main
```

File: .github/workflows/conan-package.yml [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+            conan_config_branch: 'master'
+            conan_logging_level: 'info'
+            conan_export_binaries: true
+        secrets: inherit
+
+    notify-export:
+        if: ${{ always() }}
+        needs: [ conan-package-export-linux ]
+
+        uses: ultimaker/cura/.github/workflows/notify.yml@5.1
```
Comment (Joeydelarago) [inline .github/workflows/conan-package.yml:48]:
```suggestion
        uses: ultimaker/cura/.github/workflows/notify.yml@main
```

**Review by Joeydelarago — COMMENTED** [review]
Update branch names

File: .github/workflows/conan-package.yml [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+            - '*.sig'
+            - 'conanfile.py'
+            - '.github/workflows/conan-package.yml'
+            - 'GitVersion.yml'
+        branches:
+            - main
+            - 'CURA-*'
+            - '[1-9]+.[0-9]+'
+        tags:
+            - '[1-9]+.[0-9]+.[0-9]+'
```
Comment (casperlamboo) [inline .github/workflows/conan-package.yml:22]:
```suggestion
            - '[0-9]+.[0-9]+.[0-9]+'
```

File: .github/workflows/conan-package.yml [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+        paths:
+            - '*.material'
+            - '*.sig'
+            - 'conanfile.py'
+            - '.github/workflows/conan-package.yml'
+            - 'GitVersion.yml'
+        branches:
+            - main
+            - 'CURA-*'
+            - '[1-9]+.[0-9]+'
```
Comment (casperlamboo) [inline .github/workflows/conan-package.yml:20]:
```suggestion
            - '[0-9]+.[0-9]+'
```

**Review by Joeydelarago — APPROVED** [review]

**Pull request merged**